catalogue/webapp/pages/item.js

```diff
@@ -184,7 +211,8 @@ const ItemPage = ({
       {pdfRendering && !mainImageService && (
         <IframePdfViewer title={`PDF: ${title}`} src={pdfRendering['@id']} />
       )}
-      {mainImageService && currentCanvas && navigationCanvases && (
+      {((mainImageService && currentCanvas && navigationCanvases) ||
```

Comment:
is `mainImageService` a necessary check anymore? Doesn't look like we use it inside this conditional.

Reply:
you're right, will remove